driver/src/contracts/stablex_contract.rs

```diff
@@ -23,8 +23,7 @@ pub struct StableXContractImpl {
 
 impl StableXContractImpl {
```

Comment:
Probably not worth changing this name. Looking forward to using ethcontracts-rs for all of this :-)